src/content/en/2020/markup.md

```diff
@@ -1132,14 +1111,13 @@ There has been a lot of [discussion](https://adrianroselli.com/2016/01/links-but
   sql_file="pages_markup_by_device.sql"
 ) }}
 
-{# TODO(analysts): Where do these "occurrences" come from? Ideally we have a single sheet to link to with the results used by this table. #}
 <figure>
   <table>
     <thead>
       <tr>
         <th>Button types</th>
-        <th>Occurences</th>
-        <th>Percentage</th>
+        <th>Pages</th>
```

Comment:
Same general problem as mentioned before and no, “pages” would be inaccurate.

Reply:
Ah this one I'm definitely wrong! 

```suggestion
        <th>Occurrences</th>
```